lib/metanorma/cli/command.rb

```diff
@@ -77,6 +78,36 @@ def collection(filename = nil)
         UI.say e.message
       end
 
+      desc "convert FILENAME", "Convert STS XML to Metanorma adoc"
+      option :output_format,
+             type: :string,
+             desc: "Output format: xml|adoc|iso|niso"
+      option :output_file, type: :string, desc: "Output file"
+      option :imagesdir,
+             type: :string,
+             desc: "For STS input only: folder with images"
+      option :input_format,
+             type: :string,
+             desc: "Input format: metanorma|sts|rfc"
```

Comment:
What does "metanorma" mean? Also, there is NISO STS vs ISOSTS that are different formats. Ping @Intelligent2013

Reply:
I got that values from `mnconvert.jar --help`